css-pseudo-classes/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>CSS Psuedo Classes</title>
+  </head>
+<body>
+  <button class="btn">I'm a button!</button>
+  <h1>Striped Table</h1>
+  <table>
+    <thead class="head">
+      <tr>
+        <td>Student Name</td>
+        <td>Course</td>
+        <td>Grade</td>
+      </tr>
+    </thead>
+    <tbody>
+      <tr class="gray">
```

Comment:
instead of applying the class of `gray` to every other `tr` element, find a way to use pseudoclasses to accomplish this behavior. Look into using a selector with a combinator [https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Combinators](url) to select `tr` elements that are direct children of `tbody`. Then add the `nth-child` [https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child](url) pseudoclass to your selector to pick every other row and style appropriately.